test/tests.js

```diff
@@ -241,8 +241,29 @@ QUnit.test('false secure value', function (assert) {
 	assert.strictEqual(actual, expected, 'false should not modify path in cookie string');
 });
 
+QUnit.test('unofficial attribute', function (assert) {
+	assert.expect(1);
+	assert.strictEqual(Cookies.set('c', 'v', {
+		unofficial: 'anything'
+	}), 'c=v; path=/; unofficial=anything', 'should write the cookie string with unofficial attribute');
+});
+
+QUnit.test('unofficial attribute with boolean value', function (assert) {
+	assert.expect(1);
+	assert.strictEqual(Cookies.set('c', 'v', {
+		unofficial: true
```

Comment:
I understand. Let's review the focus of this PR then:

1. Allow the `SameSite` attribute to be passed for those willing to use it at their own risk. js-cookie won't support it explicitly so we won't document how it works, until the attribute is amended to the official RFC 6265 spec.
2. Allow the same code to be generic in a way that allows people to experiment using custom attributes that are different than the `SameSite` attribute.
3. Reduce the amount of code we need to add in js-cookie and the amount of behavior we need to support

For the number **1** we can just add support for the `{ unnoficial: String }` structure. For the number **2** we need to add `{ unnoficial: true }`, however that will conflict with number **3**.

What if we build number **1** and leave number **2** out for now in a way that we can achieve number **3** without any issue? Later we can implement number **2** if there's enough requests.

Thoughts?